packages/flutter/test/widgets/overlay_test.dart

```diff
@@ -113,7 +113,7 @@ void main() {
         ' ╎ │   size)\n'
         ' ╎ │ constraints: BoxConstraints(w=800.0, h=600.0)\n'
         ' ╎ │ size: Size(800.0, 600.0)\n'
-        ' ╎ │ alignment: AlignmentDirectional.topStart\n'
+        ' ╎ │ alignment: top-start (directional)\n'
```

Comment:
yeah I don't think it is a big win. Switched to normalizing how enum properties were displayed instead of changing the alignment classes to match enums.
now this line stays as AlignmentDirectional.topStart
The only change is standard non-directional alignment constants are displayed like enum properties are now displayed so
topStart for Alignment.topStart.
Deprecated the existing display of enums as dash separated as it wasn't worth the cost in diverging debug values from the values users write in code.